Manifests/ManagedPreferencesApplications/com.secondsonconsulting.renew.plist

```diff
@@ -0,0 +1,543 @@
+					<key>pfm_title</key>
+					<string>"No Deferrals Remaining" Button Text</string>
+					<key>pfm_description</key>
+					<string>>What do you want your deferral button to say when it's greyed out in Aggressive Mode? Maps to SwiftDialog option: --button1text</string>
+					<key>pfm_description_reference</key>
+					<string>https://github.com/SecondSonConsulting/Renew/wiki/OptionalArguments</string>
+				</dict>
+				<dict>
+					<key>pfm_name</key>
+					<string>RestartButtonText</string>
+					<key>pfm_type</key>
+					<string>string</string>
+					<key>pfm_title</key>
+					<string>Restart Button Text</string>
+					<key>pfm_description</key>
+					<string>What do you want your "Restart Now" button to say? Maps to SwiftDialog option: --infobuttontext</string>
+					<key>pfm_description_reference</key>
+					<string>https://github.com/SecondSonConsulting/Renew/wiki/OptionalArguments</string>
+				</dict>
+				<dict>
+					<key>pfm_name</key>
+					<string>ShowDeferralCount</string>
+					<key>pfm_type</key>
+					<string>boolean</string>
+					<key>pfm_title</key>
+					<string>Show Deferral Count</string>
+					<key>pfm_description</key>
+					<string>If true, an additional sentence is added to your Normal Mode dialog windows indicating how many deferrals the user has remaining. Omit this key if you don't want the extra information.</string>
+					<key>pfm_description_reference</key>
+					<string>https://github.com/SecondSonConsulting/Renew/wiki/OptionalArguments</string>
+				</dict>
+				<dict>
+					<key>pfm_name</key>
+					<string>NotificationIcon</string>
+					<key>pfm_type</key>
+					<string>string</string>
+					<key>pfm_title</key>
+					<string>Notification Icon</string>
+					<key>pfm_description</key>
+					<string>This is the icon included on the right side of the Dialog Notification. Maps to SwiftDialog option: --icon</string>
+					<key>pfm_description_reference</key>
+					<string>https://github.com/SecondSonConsulting/Renew/wiki/OptionalArguments</string>
+				</dict>
+				<dict>
+					<key>pfm_name</key>
+					<string>NotificationIconDarkMode</string>
+					<key>pfm_type</key>
+					<string>string</string>
+					<key>pfm_title</key>
+					<string>Notification Icon DarkMode</string>
+					<key>pfm_description</key>
+					<string>This is the icon included on the right side of the Dialog Notification. Use this key if you have to differentiate between light and dark mode icons. Maps to SwiftDialog option: --icon</string>
+					<key>pfm_description_reference</key>
+					<string>https://github.com/SecondSonConsulting/Renew/wiki/OptionalArguments</string>
+				</dict>
+				<dict>
+					<key>pfm_name</key>
+					<string>MessageIcon</string>
+					<key>pfm_type</key>
+					<string>string</string>
+					<key>pfm_title</key>
+					<string>Message Icon</string>
+					<key>pfm_description</key>
+					<string>The icon used in your Normal and Aggressive Mode dialogs. Use "none" to disable.  Maps to SwiftDialog option: --icon</string>
```

Comment:
Please clarify that this is the path to the icon, and what formats are accepted. Also applicable to the next three subkeys.

Reply:
Thanks for all of the feedback, Ari.

A style question for you: Since these keys are mapped directly to options in the SwiftDialog project, would it be appropriate for me to use the URL to their documentation for these specific keys? Or should I keep the URL to my own project in the `pfm_documentation_url` fields and put a link to their project in the `pfm_description_reference` fields?

Reply:
I think that if the documentation is reproduced on your project as well as at SwiftDialog then URLs to _your_ project would be less confusing for new users. If you use yours then I'm not sure that another link to the original documentation will be necessary.

Links are not suitable for the `pfm_description_reference` tag, and also, keep in mind that at present the tag is only displayed in ProfileCreator when the `pfm_description` tag is not present and is ignored completely by iMazing Profile Editor.

Reply:
understood. All entries have been changed to `pfm_documentation_url` and they all point to my own project (I have plenty of links to SwiftDialog in my documentation)

`pfm_description_reference` keys have been removed entirely from my keys (I left those entries related to the template)